aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -603,18 +615,15 @@ class AztecText : EditText, TextWatcher {
 
                 private fun replaceImage(drawable: Drawable?) {
                     it.drawable = drawable
-                    refreshText()
+                    post {
+                        refreshText()
+                    }
                 }
             }
 
-            /*
-             * Following Android guidelines for keylines and spacing, screen edge margins should
-             * be 16dp.  Therefore, the width of images should be the width of the screen minus
-             * 16dp on both sides (i.e. 16 * 2 = 32).
-             *
-             * https://material.io/guidelines/layout/metrics-keylines.html#metrics-keylines-baseline-grids
-             */
-            val width = context.resources.displayMetrics.widthPixels - DisplayUtils.dpToPx(context, 32)
+            // maxWidth set to the biggest of screen width/height to cater for device rotation
```

Comment:
Hey @theck13 , thanks for picking this up for review!

Actually no, it's not a typo. The parameter in `imageGetter?.loadImage` is actually about the maximum image wanted to be returned. I renamed the variable with b8bfe38 to better match that meaning. Thanks!